packages/mux-player/src/index.ts

```diff
@@ -179,6 +179,34 @@ function getProps(el: MuxPlayerElement, state?: any): MuxTemplateProps {
   return props;
 }
 
+MediaErrorDialog.formatErrorMessage = (error: { code?: number; message?: string }) => {
```

Comment:
question: any tradeoffs you see on subclassing `MediaErrorDialog` with a `formatErrorMessage` vs. modifying `MediaErrorDialog`'s `formatErrorMessage`? One pro of this approach: we automatically get granular error messaging for all themes. One con of this approach: on the off chance that folks have `<mux-player>` and some other player on the same page, folks might get unexpected results. I suspect any perf etc. considerations are negligible (only a small amount of addititonal code required for subclassing). Any others on either side you can think of?

Reply:
we put a pin in this. we will go forward with this static override for now but maybe this could be avoided by having the end user message come from the custom media element directly?

Reply:
ah we (or at least I) actually said we would be mostly fine with this approach with one small addendum - in the `else` case, we would use the base impl of `formatErrorMessage()`. Something like:

```ts
const baseFormatErrorMessage = MediaErrorDialog.formatErrorMessage;
MediaErrorDialog.formatErrorMessage = (error: { code?: number; message?: string }) => {
  if (error instanceof MediaError) {
    // ...
  }
  return baseFormatErrorMessage(error);
};
```

Reply:
yes, good catch. I somehow thought it already did that.